_posts/blog/2016-12-19-running-on-fedora-25.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Running Accumulo on Fedora 25"
+author: Christopher Tubbs and Mike Miller
+reviewers: 
+---
+
+Apache Accumulo has been available in [Fedora] for the last few releases.
```

Comment:
For the last few release of Fedora or Accumulo?   I assume Fedora.   I was thinking of the following sentence, but I am not sure I like it.

    Apache Accumulo 1.6.X was initially packaged for [Fedora] 2Y.   

I was trying to make a sentence that was more precise without being redundant or wordy.